todoapp/src/components/todo.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from "react";
+import {
+  Container,
+  TextField,
+  List,
+  Card,
+  CardContent,
+  colors,
+  Box,
+  Typography,
+  Button,
+  Stack,
+  IconButton, useTheme, useMediaQuery
+} from "@mui/material";
+import EditIcon from "@mui/icons-material/Edit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import background from "../images/back.jpg";
```

Comment:
instead of back.jpg, give it a meaningfull name. 
Also put the images in in src/assets